tests/integ/modin/index/test_index_methods.py

```diff
@@ -14,8 +14,71 @@
     assert_index_equal,
     assert_series_equal,
     assert_snowpark_pandas_equals_to_pandas_without_dtypecheck,
+    eval_snowpark_pandas_result,
 )
 
+TEST_DFS = [
+    native_pd.DataFrame(),
+    native_pd.DataFrame({"col1": [1, 2, 3], "col2": [3, 4, 5], "col3": [5, 6, 7]}),
+    native_pd.DataFrame(
+        data={"col1": [1, 2, 3], "col2": [3, 4, 5]},
+        index=native_pd.Index([[1, 2], [2, 3], [3, 4]]),
+    ),
+    native_pd.DataFrame([1]),
+]
+
+NATIVE_INDEX_TEST_DATA = [
+    native_pd.Index([], dtype="object"),
+    native_pd.Index([[1, 2], [2, 3], [3, 4]]),
+    native_pd.Index([1, 2, 3]),
+    native_pd.Index([3, np.nan, 5], name="my_index"),
+    native_pd.Index([5, None, 7]),
+    native_pd.Index([1]),
+    native_pd.Index(["a", "b", 1, 2]),
+]
+
+NATIVE_NUMERIC_INDEX_TEST_DATA = [
+    native_pd.Index([], dtype="int64"),
+    native_pd.Index([1], dtype="float64"),
+    native_pd.Index(
+        [1, 2, 3, 2, 3, 5, 6, 7, 8, 4, 4, 5, 6, 7, 1, 2, 1, 2, 3, 4, 3, 4, 5, 6, 7]
+    ),
+    native_pd.Index([1.1, 2.2, 1.0, 1, 1.1, 2.2, 1, 1, 1, 2, 2, 2, 2.2]),
+    native_pd.Index([1, 3, 1, 1, 1, 3, 1, 1, 1, 2, 2, 2, 3], name="random name"),
+    native_pd.Index(
+        [True, False, True, False, True, False, True, False, True, True], dtype=bool
+    ),
+    native_pd.Index(
+        [
+            "a",
+            "b",
```

Comment:
do you have similar test as what you have fixed? can you run this with jenkins? in fact, for when sort = True, and the result have a tie it is probably better to always jsut compare with the hand coded expected result

Reply:
Yeah, I copied over the jenkins test here